src/components/navbar/index.js

```diff
@@ -2,42 +2,76 @@ import React from 'react';
 import styled from 'styled-components';
 
 const NavWrapper = styled.header`
-  background-color: lightpink;
-  padding: 10px;
-  color: black;
+  position: fixed;
+  background: #48C6DC;
```

Comment:
after pulling from the frontend branch, you will see a theme provider where you can use colors like this:
```
${(props) => props.theme.colors.white};
```
Can you replace your instances of colors with this?